addon/components/scatter-chart.js

```diff
@@ -258,11 +263,15 @@ const ScatterChartComponent = ChartComponent.extend(LegendMixin, FloatingTooltip
     var legendData = this.get('groupedData').map(function(d, i) {
       var name = d[0].group;
       var value = d.length === 1 ? d[0] : null;
+      // Get the color of the group. Because they are in the same group, they
+      // should share the same color, so we only need to get the color of the
+      // first object and pass to the function
+      var color = getGroupColor(d[0], i);
       return {
         label: name,
         group: name,
-        stroke: getGroupColor,
-        fill: displayGroups ? getGroupColor : 'transparent',
+        stroke: color,
```

Comment:
@jordansmith42 : It takes either a color string or a function so this change should be fine here:
https://github.com/Addepar/ember-charts/blob/master/addon/mixins/legend.js#L208-L213